README.md

```diff
@@ -1,5 +1,4 @@
-locus
-=====
+# locus
 
```

Comment:
Most of the changes to this file are the result of changes to `pandoc` itself, which is used to generate it.